scenarios/kubernetes_e2e.py

```diff
@@ -724,6 +756,9 @@ def create_parser():
         '--kops-state', default='s3://k8s-kops-jenkins/',
         help='Name of the aws state storage')
     parser.add_argument(
+        '--kops-state-gce', default='gs://k8s-kops-gce-jenkins/',
```

Comment:
I'm not sure if we have a suitable bucket - I opened https://github.com/kubernetes/test-infra/issues/5120 to track